.gitignore

```
@@ -1 +1,2 @@
 kas-local.yaml
+**/**/__pycache__
```

```suggestion
**/__pycache__
```
This should have the same effect.